packages/client/src/Call.ts

```diff
@@ -1221,6 +1217,84 @@ export class Call {
     return this.publisher?.updateVideoPublishQuality(enabledRids);
   };
 
+  private handleOnTrack = (e: RTCTrackEvent) => {
+    const [primaryStream] = e.streams;
+    // example: `e3f6aaf8-b03d-4911-be36-83f47d37a76a:TRACK_TYPE_VIDEO`
+    const [trackId, trackType] = primaryStream.id.split(':');
+    this.logger('info', `Got remote ${trackType} track:`);
```

Comment:
This is probably a bad merge. `handleOnTrack` now lives in `Subscriber`.

Reply:
uh, thanks for catching that fixed